File: backend/src/main/java/com/grouply/backend/post/project_post_position/dto/ProjectPostPositionDTO.java
package com.grouply.backend.post.project_post_position.dto;

import com.grouply.backend.project.project_member.ProjectPosition;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class ProjectPostPositionDTO {

    private Long id;
    private Long projectPostId;
    private ProjectPosition position;

}

File: frontend/src/features/feed/models/ProjectPostPositionDto.ts
import type { ProjectPosition } from "../../../shared/models/project/ProjectPosition";


export class ProjectPostPositionDTO {

    id: number 
    projectPostId: number
    position: ProjectPosition

    
    constructor(id: number, projectPostId: number,position: ProjectPosition) {
        this.id = id;
        this.projectPostId = projectPostId;
        this.position = position;
    }
}